### ISSUE TITLE
Hey guys, I'm Mai

### ISSUE THREAD

**Issue by mnie6637** [TheBreakingGoodProject/Dalyell-Projects#2]
Hello everyone!
I'm currently a first-year student studying a bachelor of science/advanced studies at USYD. I'm majoring in applied medical science and medicinal chemistry, and am interested in going into the pharmaceutical field after graduating. I chose to work on this project, with E$$ENTIAL MEDICINE$, as it aims to help the public learn about and research the very medicines they use in their own lives. I feel that with the growing distrust in scientific research in our society, emphasising individual learning surrounding medicines is something sorely needed, and I am hoping to be able to gain insight into how scientists can help the public through this program. I'd also love to experience working with a group of collaborators and mentors, to hopefully create a meaningful end product :)

**Comment by yaelago** [comment]
Welcome Mai!
It's wonderful to have you on broad. Thanks for your insightful introduction.

**Comment by mnie6637** [comment]
After a bit of a delayed start to the process, I've begun the challenges for the E$$ENTIAL MEDICINE$ project. After jumping straight into the News Flash challenge, I was immediately a bit confused. My impressions of the challenge (from reading the Breaking Good website, and hearing the description of the challenge from coordinators) were that it was the start of the research process, and I should pick a drug to research. So my immediate thoughts upon seeing it ask for my personal knowledge of news in medicines were, "this challenge is not for me". I realise that this is mainly due to my own preconceptions of the challenge, but it made me think that maybe keeping this as the first challenge without a preliminary introduction to it may put some people off from completing the challenges, as they feel their initial knowledge base is inadequate.

Still confused, I thought I needed to pick a medicine and complete the challenge, so I chose Levonorgestrel to do a quick google on, to simulate me being a normal citizen hearing things on the news. One of the first news articles, [https://www.dailymail.co.uk/news/article-9367287/AstraZeneca-Women-likely-develop-blood-clots-taking-contraceptive-pills-vaccine.html](url), was striking as it was a negative response to a bad side effect, and I thought this is something the public would focus on, due to its 'click-baitey' heading. To be honest, seeing a header and reading an article like that, seeing that nothing was done to regulate the drug and its possible harmful side effects, did make me question the governments ability to control potentially dangerous medicines. From this I saw how easy the media could change the public's opinions of the scientific/pharmaceutical industry to something negative and distrustful. The questions that the News Flash challenge asked helped me facilitate this understanding of public opinion.

**Comment by mnie6637** [comment]
By the start of challenge 2, the Price Hike, I understood more of what was going on. The example given in the intro was very good for personal motivation, as it showed me what type of things to look for and how price increases are a serious problem in essential medicine distribution. While the information given was very useful, I started to notice how user friendliness is a bit of a problem in the project, as the interface is a bit drab and unclear. This could be an area for improvement. 

The medicine I ended up choosing was doxycycline, which was linked to my own personal interests having previously used the antibiotic. The instructions given and the tips to aid with searching were very clear. My first few searches did not give any usable results, as most websites ended up talking about overseas price hikes, eg. [https://www.fiercepharma.com/manufacturing/doxycycline-price-hikes-make-antibiotic-too-expensive-for-pets-and-vets](url), or they were non-trustworthy websites, which I judged based on the given instructions to look for gov, edu etc. domains. I was struggling to find relevant information by searching keywords such as "doxycycline price hike australia" or "price change" or "price increase", and was getting increasingly frustrated.

While researching, I found that a positive by-product of the tiring research i was doing was that I was passively learning a lot of medical information from various sources. Just by skimming through articles and government databases (eg. [https://www.nps.org.au/medicine-finder/apo-doxycycline-tablets](url)), I read through a bunch of good information on the product; the proper use and side effects of doxy, that it can be sold under the brand name APO-Doxycycline, and that a lot of medicines have doxycycline monohydrate as the active ingredient. This indicates that the E$$ENTIAL MEDICINE$ project may actually help citizens passively learn a wide array of medical information past that of just prices. Considering this, I also began contemplating the fact that if one is not being mindfully selective of which sources are accurate, one can also passively learn incorrect details.

I was still attempting to find trustworthy, related sites, and one place I found some useful links to was the bottom of the Australian Health Thesaurus site, linking the Australian Register of Therapeutic Goods website [(https://www.tga.gov.au/australian-register-therapeutic-goods](url)) among others. While I did not find information on price hikes, a link on the TGA search website led me to information of the Black Triangle Scheme, which encourages doctors and patients to report adverse reactions to new medicines as a form of citizen science itself. I thought this was surprisingly relevant, and I was also wondering why I had never heard of it before. 

I was getting increasingly frustrated about not finding information, so I used the advice given on the price hike survey and changed my approach. I moved from using Bing to using Google search :). And this helped immensely, the first page was relevant to australian price hikes, ([https://www.health.harvard.edu/blog/why-many-generic-drugs-are-becoming-so-expensive-201510228480](url)) and said from 2012 to 2013 the "cost of doxycycline increased from 6 cents to $3.36 per pill" and gave a reason for this price hike as being due to a manufacturer closing.  This fact was then corroborated with some other websites I found. The next PBS part of the challenge was the easiest part to complete, although I think it would be a good idea to specify the price for what unit in that question of the survey, eg. price for a pill or price for 100g?

**Comment by mnie6637** [comment]
Challenge 3 for doxycycline was the hardest challenge out of the set. Initially, I was unsure whether we were supposed to start from the initial applicant holder we picked or if we should research all of them, because why would we only choose one if we're supposed to research the full timeline of the drug? I ended up using it as a starting keyword only, but the information I found barely mentioned the other applicant holders of doxy and I couldn't fit them into my timeline. The instructions for this task could have been a bit clearer. I was also unsure whether to use international sources or not, coming from the previous Price Hike challenge which was specifically for Australia. 

I struggled a lot with the research for this drug, and ended up using some untrustworthy(seeming) sources such as [http://www.chm.bris.ac.uk/motm/doxycycline/doxyh.html](url) and Wikipedia. Although, that ended up being rather helpful as the same information, regarding Pfizer being the initial developer of the drug, was corroborated often throughout these websites. From the references in the Wikipedia article, I found an academic paper by a scientist who talked about the research process of tetracyclines, and how a research effort led by a Pfizer scientist led to the development of doxy ([https://www.researchgate.net/publication/51907943_The_history_of_the_tetracyclines](url)). I found academic papers to be most useful in this challenge, but they come with the disadvantage that they are typically hard to read and/or access (some publishing websites require making an account). This is not really viable for a regular citizen to access. Additionally, the 'event finding' stage of this challenge required some research into law reports for court cases between companies, which was definitely literature out of my comfort zone. From reading it laboriously though, I managed to find that there was a patent infringement that resulted in many other generic forms of doxycycline being made by various companies. I had no idea how to input all of this information into the challenge survey, however, so I feel like a lot of my research efforts were lost as I could not figure out how to format it to answer the questions given.

**Comment by mnie6637** [comment]
### Medicine 2: Clotrimazole

News Flash:
I learnt from my first attempt, and this time I used my own knowledge of the Pfizer vaccine to fill out the form. It may not be very relevant as an essential medicine, but it is certainly what I've been hearing a lot of in the news these days. Not much learning was done in this challenge, but it was useful to synthesise one's own knowledge of a subject. 

Price Hike:
I first searched on google (tried and trusted) the terms 'clotrimazole price hike'. Scanning the website descriptions on google showed a lot of references to 'suspicious price increases' of general medicines, so I knew there was a price hike to be found. 

From this initial search, came up with an American article ([https://khn.org/news/climbing-cost-of-decades-old-drugs-threatens-to-break-medicaid-bank/](url)) that had the info "medicaid spent $1,459,161.71 extra in 2016 on clotrimazole" in a database. I wanted to find more relevant information and also check the trustworthiness of the site, so the next step I took was to look at the references used in the website. I found [https://www.arnoldventures.org/work/drug-prices](url), which seemed to have relevant journalism on price hikes, but nothing on clotrimazole specifically. Another website from the initial google search was [http://www.pharmabiz.com/NewsDetails.aspx?aid=129007&sid=1](url), which revealed "The prices of anti-infective APIs such as [...] clotrimazole [...] have hiked by 24% to 38% from January to April this year". It was very recent which was great and it talked about COVID related price hikes in 2020, but I decided this wasn't a very trustworthy website as I could not find any information on the author. 

So far the process has been; 
1. searching up basic search terms
2. picking a relevant-seeming website
3. scouring it for information
4. searching for references from the website
5. rinse and repeat for the new sites

I think this is a good starting point for internet research, but sometimes it doesn't come up with results, in which case I then went back to google and tried new search terms (eg. included companies who manufacture clotrimazole with 'price increase'). I happened to find this legal document [https://www.courthousenews.com/wp-content/uploads/2020/03/Generics.pdf](url). It is an American file, but has a lot of useful trends for a wild variety of medicines including clotrimazole. I thought this was very lucky and would also help with the other medicines I would do for the challenge. I believed it was trustworthy as it contains evidence used in court; however I have no idea if this information is actually right (searched up whether the evidence was found wrong on [pacermonitor.com](url) but I did not find anything). However another court finding did corroborate the information [https://www.nj.gov/oag/newsreleases19/Teva-Complaint-for-CT-District-Court_FINAL-UNREDACTED.pdf](url). The information I found on pg 315 was;
- clotrimazole was manufactured/sold by Actavis, Glenmark, Hikma, Sandoz and Taro
- In June 3, 2014 Taro increased prices on Clotrimazole due to an unethical agreement with Teva; Zydus also a company involved
- "No shortages or other market features can explain Defendants' price increases during relevant period"
- Prices remained artificially high after Summer 2014
- A graph showing the price hike; 
![image](https://user-images.githubusercontent.com/80505129/112572240-e15c3d80-8e3d-11eb-9df4-b13056adac5b.png)

This was a very lucky find, and gave me a lot of useful information. The only issue was this was entirely based in America, and I was looking for price increases in Australia. The previously mentioned companies do not actually sell in Aus, and my attempts at adding 'australia' as a search term did not end up with any results. I attempted to search 'drug price hike increase australia "clotrimazole"' to make sure my searches mentioned it, and I found from an Australian website the WHO Guideline of Country Pharmaceutical Pricing Policies which stated; 
"In El Salvador 2006, there was a markup of 179% LPG clotrimazole cream wholesale, and a retail mark up of 367%"
I also found a uni research paper [http://unsworks.unsw.edu.au/fapi/datastream/unsworks:10005/SOURCE02?view=true](url) which showed that "In 2005 in Vietnam, the public sector prices were 23.23 VND median while the public procurement price was 4.30 VND". I decided I had found no more useful information relevant to Australia, decided to stop there after over an hour of searching and entered information from America into the survey. 

I think using Google's must include function (" ") helped me greatly in this task, and also I found myself to be developing more of a method to my research as defined above. I  found myself to be getting better at isolating information when skim reading website, using the ctrl+f function, and I was definitely employing more focused learning techniques as opposed to the 'passive' learning I used in my first set of challenges. Even so, this was a greatly stressful process and being surrounded by so much information was a little overwhelming.  

Circle of Life:
This only took about 15 minutes compared to the previous challenge's 1-2hrs, so I felt like I was breezing through it. From the E$$ENTIAL MEDICINE$ database, it was good to see Taro was a company I had encountered in previous research, so I knew I was on the right track.

I started by searching "clotrimazole discovery" on Bing from which I found [https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/jam.12554](url) and [https://books.google.com.au/books?id=FB_2CAAAQBAJ&pg=PA109&redir_esc=y#v=onepage&q=clotrimazole&f=false](url), which told me clotrimazole was discovered in 1969 and then was launched in 1973. I couldn't find any more relevant information, so after 10 minutes I switched to Google using the same search terms and found exactly what I needed in [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4724869/#:~:text=Clotrimazole%2C%20first%20synthesized%20by%20Karl,under%20the%20brand%20name%20Canesten.](url), which told me who originally owned it. Then I searched "taro clotrimazole" and found [https://www.nytimes.com/1996/01/21/nyregion/pharmaceutical-is-a-leader-in-generics.html](url), which told me how the switch from companies happened. 

This streamlined research I ascribe mostly to luck or the general availability of this information, not my own research skills, but it was very nice to find the information so quickly. I find that so far in general, the research process has left me feeling frustrated and tired, so this was a nice refresher. I think making the research process a bit more orderly would help my feelings of frustration for future researching.

**Comment by mnie6637** [comment]
### Medicine 3: Entecavir

News Flash:
I had heard of general medicine shortages in Fiji due to the effects of COVID, so I entered that as my response. After doing this challenge a few times, I'm realising I don't hear about medicines much in the media, apart from COVID-19 topics. I think this recognition of my lack of knowledge is the first step into actively seeking to learn about current issues in medicine.

Price Hike:
As entecavir was not familiar to me in the slightest, I did a quick search of the name to get an idea of what it is, and found it's an antiviral medicine used to treat Hep-B,currently sold under brand name Baraclude by Bristol-Myers Squibb. I then dived into my research, googling 'entecavir price hike australia'. The first result, [https://gabionline.net/layout/set/print/Generics/Research/Costs-and-prices-of-entecavir-to-treat-Hepatitis-B](url), told me that in 2015 Entecavir was no longer under patent protection in China, Brazil, South Africa and the US, with European patent expiry in 2017. This was corroborated in the next website I checked, [https://www.prnewswire.com/news-releases/china-entecavir-investigation-market-report-2018-2022---market-size-competitive-landscape-price-by-manufacturer--market-outlook-300743012.html ](url), which also told me that entecavir was originally developed by BristolMyersSquibb and entered the market in 2005. The website also told me a subsidiary of this company, SinoAmerican Shanghai Squibb Pharmaceuticals Ltd, started selling the drug in China in 2006. 

The first mentions of cost were found in the third website I checked, [https://www.sciencedaily.com/releases/2015/04/150422121726.htm](url), which quotes, "Scientists at the University of Liverpool have demonstrated that a drug for treating hepatitis B virus (entecavir) could be mass-produced for only £24 ($36) per person per year, versus the current UK NHS price of £4,600, and the US price of over $15,000". This was trustworthy information as it cited a university research paper, and also told me that as an article published in 2015, the price of entecavir may have actually dropped in price over time. No other information concerning price hikes were come across from my first initial searches, so I checked the courthouse document from the clotrimazole challenge (see above), and although they provided information on enteclavir, I found no mention of price increase. It mentioned unethical business tactics with Teva (and the fact that the patent was invalidated for ‘obviousness’ in 2014) but nothing on prices for consumers. 

As so far I had not come across any evidence to suggest a price increase, I tried to confirm my theory of a price decrease and searched 'current us prcie for entecavir’. The search results were mainly the same results as before, so I changed the search to 'entecavir price', and found [https://www.goodrx.com/entecavir](url), which is an American site for pharmaceutical price comparisons. I found the monthly cost for entecavir to be ~$45, which (given the medicine could be subsidised by the government) was not a significant decrease from the university report above in 2015. Hence none of what I found let me come to the conclusion that the price had changed, although this was a difficult decision to make and I felt I had just not researched enough. But after trying to search many variations of 'price increase' to no avail, I felt it useless to continue. 

Talking about the survey; I selected no price hike on the challenge website, however when asked for information about changes in the price of entecavir, I could not enter in my research as it didn't fit the mold of the questions (eg. why has the price changed - entecavir hasn’t changed yet but may in the future). This may be useless information to researchers, but I do feel that the set up of the survey is done in a way that prevents the full depth of research to be presented. In retrospect, this may be due to the fact that it is easier to sort through information this way, so it may not be a problem. I also find however that some things seem to make it harder for people to compile the information from these surveys, eg. the price of the PBS drug not being in consistent units, or for specific doses/the price of the API itself. 

Circle of Life:
All the information I had to answer this question I had found in the previous challenge, which was helpful. I think over time I have learnt to compile information I come across that I know will be useful for later, which has quickened the research process a lot. I believe I can do this because I am familiar with the questions I will come across. 

The problem with this challenge was the fact that the company that initially produced the drug enteclavir, BMS, was the only one listed in the database. This made it impossible to answer the questions in the survey accurately, as the 'beginning' of the drug's circle of life was also the 'end', and there was obviously no event that passed the drug from the same company to itself. I could however have mentioned the end of the patent in 2014, and the other companies that currently sell enteclavir (eg. Mylan in Australia), but there was no appropriate place to enter this information in the survey. This was a bit disappointing and I think makes the survey hard to use to get the full grasp of the drugs' lifetime.

**Comment by mnie6637** [comment]
### Medicine 4: Albendazole

News Flash:
I recalled seeing some information on an ‘organic’ alternative for antihistamines; not sure if this actually counts as a medicine, but it was backed by some doctors/academics, so I used it for this challenge. It's getting a lot harder to think of new things I've heard or seen relating to medicines, but at the same time I don't think this challenge is the main point of our Dalyell project study.

Price Hike:
I selected the drug albendazole, which I had never heard of. I googled it and I learnt it was an 'anthelmintic' drug, meaning it is used to kill parasitic worms in the body. I followed the same process as before (starting with broad searches and narrowing down each search path as you go, kind of like a branched tree of information pathways). My first search - 'albendazole price hike' - came up with “In the years following albendazole's price hike in 2011”, telling me immediately the details of what happened (yes there was a price hike) and the fact that it was in 2011. Looking at the website ([https://www.npr.org/sections/goatsandsoda/2017/12/11/567753423/why-a-pill-thats-4-cents-in-tanzania-costs-up-to-400-in-the-u-s](url)), I was given a lot of useful information;
- Mentions price discrepancies between countries; eg. in Tanzania the two pills needed to wipe out a parasite is 4c, but in US it would be $400. 
- In the US it is only sold by Impax Laboratories
- Patent ended years before 2017, pharmaceutical companies in developed countries like America have just been slow to jump in the market (less potential for profit as only few cases are affected)

I didn't find anything about specific pricing on that page, so I moved onto the next intial search result, [https://www.frontiersin.org/articles/10.3389/fsoc.2020.540478/full](url), which ended up being incredibly helpful. Not only did it tell me the exact years, costs and companies involved with the price hike (“Between 2010–2015, the AWP of Albenza increased by 3,299%, from $5.92 per 200 mg tablet in 2010 to $201.27 in 2015 (Alpern et al., 2016)”), it also explained the reasons behind it. The following paragraph summed up exactly what I had found to be a trend with some medicines;

> “Price-control depends primarily on the entry and persistence of generic products following the expiration of the market exclusivity period granted to the manufacturer of the brand name drug. Unfortunately, barriers to generic entry are common, allowing off-patent drugs like albendazole to remain relatively expensive despite having been marketed in the US for decades.”

Note: I tried to access the Alpern et al., 2016 document to see if the information was trustworthy, but I could not access it due to the report publishing website requiring an account.

The website also explores the timeline of the prices of the drugs, which was extremely useful in answering the 'why' behind the price hike, as well as helping with the next Circle of Life challenge. In summary,

> "In 2010, CorePharma acquired the marketing license for Albenza (albendazole) from GlaxoSmithKline (GSK) and then sold the drug to Amedra Pharmaceuticals, a private equity firm. Amedra then purchased the primary competitor in the US market, mebendazole. Between 2010–2015, the AWP of Albenza increased by 3,299%, from $5.92 per 200 mg tablet in 2010 to $201.27 in 2015 (Alpern et al., 2016). In 2015, Impax Labs (now Amneal Pharmaceuticals Inc.) acquired Amedra. This led to subsequent increases in the price of Albenza, eventually landing on its current average wholesale price of $291.21 per tablet"

However the site gave a lot more information as well on the Average Wholesale Price (AWP) over time, and the reasons behind the change. This information provided me with enough information to fill out the surveys for the challenges; the website itself also seems very reliable, with trustworthy annotations, and it presents an unbiased view of the situation by exploring the weaknesses and strengths of the evidence used. (Eg. "Although the AWP is not a good measure of the price actually paid for a drug, it can translate to high out-of-pocket costs for patients—particularly the uninsured.")

I still wanted to see if I could find Aus specific information, so I added ‘australia’ to search terms. I found
[https://blog.petrieflom.law.harvard.edu/2018/10/17/the-rotten-u-s-antiparasitic-drug-market/](url) which stated;
“In the UK it costs only $0.66 per pill and in Australia it is $0.18 per pill”, showing the price hike of albendazole may be a US centralised issue.

For entering my research into the survey, there was no place to enter the subsequent price decreases after the initial price hike → I feel like this is important information that would give scientists access to the full timeline of a drug’s price, rather than just price hikes, so it could be an element of the interactivity of the website that could be changed.

Circle of Life:
Most of the information I needed I had already found in the previous challenge, which was extremely helpful. I confirmed that GSK was original owner by a quick wikipedia search (typically not a trustworthy site, but it linked the proper patent which confirmed the information; US3915986A - Methyl 5-propylthio-2-benzimidazolecarbamate - Google Patents). Since I had already seen there was an acquisition in 2010, I searched for further details with the terms ‘albenza acquisition gsk’ and ‘albenza acquisition gsk cost’. I found [https://www.pharmaceutical-business-review.com/news/corepharma-signs-agreement-with-gsk-to-acquire-dexedrine-us-ndas_251010/](url), which detailed the companies and drugs involved, but not the pricing for the business deal. I checked the GSK annual reports, but there was no mention of an acquisition, only the company’s humanitarian aid in donating albendazole to undeveloped countries. I could not find prices of acquisitions anywhere, so I just included the two events that I had read about in the article from the previous challenge. I think taking notes throughout the research process has really been helpful to me, as I can refer to the past information and resources I've come across, that I may have not considered useful at the time. 

Related to this, I’m finding that as I learn what to look for (focus my searches essentially) I am doing less of the passive learning I did in the first few challenges. It is more time efficient, and I am getting quicker results that are useful for the survey, but am I learning as much? Also may result in carelessness when researching; going faster, ignoring some parts of the texts being read.

**Comment by mnie6637** [comment]
### Medicine 5: Artemether/Lumefantrine

News Flash:
Honestly I was out of ideas for this last one :( This says something about the accessibility of information about medicines, especially to people of the younger generation like me who may not consume news through the traditional channels (tv, newspapers) but rely on internet news and social media. It’s a bit worrying that honest and important scientific journalism isn’t promoted nearly in the same way as (equally important) contemporary issues surrounding social injustice in the media. 

Price Hike:
Artemether+lumefantrine is sold under trade name Coartem, and is used to treat certain kinds of malaria. It is an ‘artemisininderative’, known for their ability to suppress immune reactions such as inflammation. 

The search terms ‘artemether lumefantrine price hike’ came up with no relevant results, most depicting various clinical trials detailing the cost and cost-effectiveness of the medicine, but not anything about general price increases. A quick look at the drug’s wikipedia page let me see that the producer of Coartem, Novartis, had actually been decreasing its price from 2001 to 2008. This information was not backed by a citation on wikipedia, so I used the search terms to narrow my search. I tried ‘coartem price decrease’, and got a wide array of results all with similar headlines, eg, Novartis cuts price of malaria drug Coartem. The following website, [https://www.mmv.org/newsroom/news/price-coartem-reduced-third-time-8-years#:~:text=On%20top%20of%20this%205,is%2036%20US%20dollar%20cents](url), details the final 5% price reduction in 2009 (the third price reduction in 8 years) as seen on wikipedia, and also mentions a new produce Coartem Dispersable, which is a sweet, water soluble form of the same medicine, designed for children. The site also mentions the “250 million antimalarial Coartem treatments” Novartis supplied “to over 45 endemic countries at not-for-profit prices and in so doing [contributing] to saving an estimated 630,000 lives”. This indicates that the price, or at least accessibility, of the medicine has decreased globally. The website [http://lists.healthnet.org/archive/html/e-drug/2006-10/msg00054.html](url) also details the more significant 36% price decrease which occurred in 2006, including the reason for which being that Novartis subsidised production by over US$10 mil a year. The reason was enunciated in [https://www.fdanews.com/articles/62909-novartis-cuts-price-of-malaria-drug-coartem](url);

> "The dramatic increase achieved in our production capacity, thanks to an improved supply situation for the natural ingredient artemisinin, provides us with an opportunity to further accelerate access to Coartem in combination with a price decrease," said Daniel Vasella, chairman and CEO of Novartis.

It was really heartwarming to hear that a company with a monopoly on a product actually chose to decrease the price of an essential medicine, helping multitudes of people beat disease, just _because they could_. (Or so it seems)

Circle of Life:
I had already learnt that Novartis had a monopoly on the product, but to show that they were the original developers I googled ‘artemether lumefantrine developer’, and this article appeared; [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6013505/](url). This directly states “The first ACT produced to recognised international standards of good manufacturing practice (GMP) was artemether–lumefantrine (Coartem®; Novartis), approved by the US FDA in April 2009 [11]”, giving me the answers for the survey.

However as I had just wrapped up my research, I came across a blog article, [https://malariaworld.org/blog/tea-against-act-david-against-goliath](url). The translation suggests that a new tea form of Artemisia (the same group of drugs as in Coartem) was just about to be proven as effective as Coartem, before WHO issued a veto on its clinical trials. 

One of the commenters had said;

> “In 2001, Novartis AG and the World Health Organization (WHO) unveiled a unique 10-year alliance to provide a breakthrough medicine against malaria. 
Novartis agreed to provide its drug, known as Coartem (artemether-lumefantrine)…The WHO reviewed requests for supplies and distributed Coartem in malaria-endemic countries”
This information is easy to find on the homepages of Novartis and WHO.
For a pharmaceutical company to have a heavy weight distributor of this kind had to be appreciated and “compensated”.
To quote THE EAST AFRICAN of 4 June 2007:
“The first consignement of Coartem worth $8 million, arrives in Kenya in May 2006. Under pressure from the WHO, Kenya adopted the drug…despite opposition by the Pharmaceutical Industry of Kenya, which protested that the drug was too expensive. A full dose of the medicine costs about $8.
Last year alone according to Kenya’s public drugs procurement agency, the Kenyan government and its development partners spent more than $ 58 million on anti-malarial drugs.
For Kenya the irony of the horrendous toll… is that the country has been recognized as the third most important center for malaria research…”

The general tone of the report and comments suggested corruption within the pharmaceutical giants, and more importantly the organisation responsible for regulating and promoting their products (WHO). For example the article suggests that WHO gets 3% on Novartis sales. This was a complete change of tone from my previous research, and while I wanted to find more information, I couldn’t find much other reliable evidence. I ended up just entering the price decreases I had found before in my survey. But this made me very wary of the information I have gathered so far in the project; to what extent are we only seeing one side of a story? Are the conspiracies behind 'Big Pharma' really true, and to this detrimental of an effect? How do we know what sources are reliable, if they may say the objective truth, but in actuality are lying by omission? 

It seems learning about research has just left me with more questions about the research process.

**Comment by amychau16** [comment]
Hey Mai!
It was interesting to read through your learning experience and to compare it with my own. Here are some aspects of your learning experience that resonated or stood out to me:

- I definitely felt the exact same way when I first started as the first challenge asked us for our own knowledge which was what I lacked. So I did make an assumption and researched on news surrounding my chosen medicine, rather than relying on my own knowledge.

- Another problem that you pointed out was the lack of user-friendliness. Although I did not find the interface to be a major problem to understand (although the design of it was quite bland), this is something worth looking further into, especially if future participants may have the same issue which makes them lose interest or get confused. 

- Building on my scientific knowledge in areas related to the medicinal uses, side effects, and brand names across different countries was definitely a key learning outcome for me. I really liked that you raised the point that as we spent time searching through many sources to find relevant information (especially on price hikes), we gained new knowledge in a passive manner. Although I do agree with how you found this to be very time-consuming and inefficient, I think that this will be beneficial as a participant given that they have some interest in medicines. However this is definitely an issue if they lack motivation or become frustrated when struggling to find relevant information.... 

- The process of spending a few hours of looking through sources and entering different search terms on Google was part of my experience as well. Although I found this quite frustrating and overwhelming, particularly for Price Hike$ and Circle of Life, I do believe that this was an important learning process that touched on our skills to interpret and analyse information (especially when we decide what piece of information is reliable and whether we need to find more sources to corroborate a piece of evidence).
However, a really important point that I felt you addressed was the fact that even though we may be accessing information from a reliable website, we are only seeing one side of the story. This raises concerns surrounding corruption within the government, pharmaceutical industry and media that many consumers including us are not aware of. This challenge may be a good way for participants to increase awareness of their surroundings. 

- I did not think of this while I was completing Price Hike$, but I liked how you pointed out that participants should not be limited to just entering price hikes but also any decreases or changes so that scientists can have a more well-rounded timeline on these price changes. In retrospect, I think this would have been useful for myself when I was completing this challenge. 
Although there was an option to complete another price hike challenge if you found more than one price change, I also think that it may be a good idea to make changes to the interface so that you can enter more than one price change within the same challenge. This would make it less confusing for the participant and allow us to enter different price changes (especially when they are across multiple brands/countries).

**Comment by mnie6637** [comment]
Hey Amy,
Thanks for your wonderful discussion of the points I made! You summarised them even better than I could have :) 

It seems we agree on a lot of these points, which is good to see since we can then isolate these aspects of the learning process to focus on. The user friendliness is something that a lot of us seemed to have issues with, so we can do some further research on that, although it is more up to the developers of the site not the participants so it may not be relevant for our 'how to guides' (but would still be a good thing to improve on).

You mentioned the point I raised about learning passively while researching, and how I saw it to be cumbersome and time-consuming in my personal experience. However, I also agree with your view that it can be beneficial to a participant as they can learn through the information exposed to them. I actually used this idea that passive learning is an integral part of the learning process, in my A4 overview. There I found that, particularly in online learning, it is the main form of engagement (Smith, D. and Smith, K. (2014) ‘The Case for ‘Passive’ Learning – The ‘Silent’ Community of Online Learners’) and thus should be an area to focus on when maximising participant learning outcomes. I think that as you said below, it is important to maximise the effectiveness of passive learning so participants do not become overwhelmed by all the information they come across in research. 

> "However this is definitely an issue if they lack motivation or become frustrated when struggling to find relevant information...."

I haven't done any research on the 'reliability of websites' aspect, but as you also concluded, I think it is an important part of the research process that could be worked upon further for our 'how to guides'.

**Comment by ShirleyZ11** [comment]
Hi Mai,

Thank you for sharing your journal entries, I really enjoyed reading through them. You have explained your research experience and learning processes in such detail. I can see that like many others, we had similar experiences in some respect, and faced similar challenges whilst completing the tasks. I emphasise with the difficulty and frustration you may have felt trying to find reliable information on certain medicines, but I think that the end result of finally finding answers was rewarding and taught me about which researching techniques were more effective than others. 

Something that I think was interesting from your entries was the use of different search engines. Personally, I have never used Bing before, but this made me consider whether some search engines are more effective than others for the research we conducted with the challenge. It could be worthwhile to see the types of information/websites that get filtered out by search engines and how the use of different ones affects one's research experience. 

I also agree with what you said about how _"learning about research has just left [you] with more questions about the research process."_ The way someone researches and learns is affected by so many different factors, like their initial understanding of a topic, current way of researching or the way that they learn. I don't think there is any one particular way that would work for everyone, so I definitely understand where you are coming from in regards to there now being more questions as we begin to dig deeper into learning processes. 

I think the experiences you have shared will become very valuable once we begin Task 3 and I can't wait to all work together on this final project :))

**Comment by leochoi2002** [comment]
Hey Mai,

After reading through your journal, it was nice to see that there were both many elements of your experience that overlapped with mine, as well as differed.

The part that most resonated with my experience, on top of the confusing  start and the drab interface, was that the survey was set up in a manner that prevents a holistic approach to research. By only asking for specific information, it was only natural for the survey to become a routine which prevented us from passively learning other information about the medicine that may have been important to note. Also, because this part about passive learning mentioned by you was something that I definitely felt but could not put into words before, I did feel quite understood.

I think this also led me to think about the nature of research, and whether this project really taught citizens how to "research" per say to extract just the necessary information. Another thing you brought up which I liked was how you said this research process may have led to more questions asked than answered. This was something I emphasised with, since I felt that there were so much more questions to be answered about the drug than just "was there a price hike". In fact, reading your journal made be think about how the challenges may have left me more overwhelmed about research than before, due to the sheet amount of questions that I was left with after completing the challenges.

One example of this was brought up in your journal, where you talked about how we are led to question who to trust when we find conflicting information online.

Thanks for sharing your experience, and I look forward to working with you for Task 3 :)
